container/state.go

```diff
@@ -278,6 +278,7 @@ func (s *State) SetRunning(pid int, initial bool) {
 	s.ErrorMsg = ""
 	s.Running = true
 	s.Restarting = false
+	s.Paused = false
```

Comment:
Hi, @thaJeztah Actually no, Now we only call the `SetRunning` after container was started. A starting container couldn't be paused.